From d1b8513bbf67db368b0727f21e2a4a1105320f22 Mon Sep 17 00:00:00 2001
From: charitardha <satya.charitardha.jayanti@intel.com>
Date: Fri, 22 Feb 2013 23:48:45 +0530
Subject: [PORT FROM MAIN}[DO NOT UPSTREAM][Background Music] Support streaming of music to WIDI/ HDMI sinks simultaneosuly along with other sinks

BZ: 63786

A new key for background music support over WIDI/HDMI is added in
AudioParameter to enable/disable this BGM wqbased on settings
set in the UI.

Change-Id: I177e12d461b7cb1f888aa5fda3cd6bb6cd3761c1
Orig-Change-Id: I6d93d3ad050b5d3e72d069ad379f55b9a5e70191
Signed-off-by: charitardha <satya.charitardha.jayanti@intel.com>
---
 include/media/AudioParameter.h    |    1 +
 media/libmedia/AudioParameter.cpp |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/media/AudioParameter.h b/include/media/AudioParameter.h
index 39e7bc1..6540849 100644
--- a/include/media/AudioParameter.h
+++ b/include/media/AudioParameter.h
@@ -50,6 +50,7 @@ public:
     static const char * const keyInputSource;
     static const char * const keyScreenState;
     static const char * const keyStreamFlags;
+    static const char * const keyBGMState;
 
     String8 toString();
 
diff --git a/media/libmedia/AudioParameter.cpp b/media/libmedia/AudioParameter.cpp
index 6fd497a..7438884 100644
--- a/media/libmedia/AudioParameter.cpp
+++ b/media/libmedia/AudioParameter.cpp
@@ -33,6 +33,7 @@ const char * const AudioParameter::keyFrameCount = AUDIO_PARAMETER_STREAM_FRAME_
 const char * const AudioParameter::keyInputSource = AUDIO_PARAMETER_STREAM_INPUT_SOURCE;
 const char * const AudioParameter::keyScreenState = AUDIO_PARAMETER_KEY_SCREEN_STATE;
 const char * const AudioParameter::keyStreamFlags = AUDIO_PARAMETER_KEY_STREAM_FLAGS;
+const char * const AudioParameter::keyBGMState = AUDIO_PARAMETER_KEY_REMOTE_BGM_STATE;
 
 AudioParameter::AudioParameter(const String8& keyValuePairs)
 {
-- 
1.7.4.1

